detox/src/devices/drivers/android/emulator/EmulatorDriver.js

```diff
@@ -82,13 +80,10 @@ class EmulatorDriver extends AndroidDriver {
     return this._emuVersionResolver.resolve();
   }
 
-  async _boot(avdName, adbName) {
-    const coldBoot = !!this.pendingBoots[adbName];
-
+  async _boot(avdName, adbName, bootPort) {
+    const coldBoot = !!bootPort;
```

Comment:
Please, give me an example when the boot **is not** cold.
I can't imagine this situation by looking at code, maybe you have an idea, @d4vidi ?